config/exercise_readme.go.tmpl

```diff
@@ -23,7 +23,6 @@ raise Exception("Meaningful message indicating the source of the error")
 
 To run the tests, run the appropriate command below ([why they are different](https://github.com/pytest-dev/pytest/issues/1629#issue-161422224)):
 
-- Python 2.7: `py.test {{ .Spec.SnakeCaseName }}_test.py`
 - Python 3.4+: `pytest {{ .Spec.SnakeCaseName }}_test.py`
```

Comment:
Track documentation will mention elsewhere that we support Python3.4+, I'm not sure it's necessary to do so here. Something like the following will suffice for this template:

> To run the tests, run `pytest {{ .Spec.SnakeCaseName }}_test.py`